examples/reference/elements/bokeh/Image.ipynb

```diff
@@ -135,9 +202,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python [conda env:earthml_v2]",
+   "language": "python",
+   "name": "conda-env-earthml_v2-py"
+  },
   "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
    "name": "python",
-   "pygments_lexer": "ipython3"
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.3"
```

Comment:
We usually clear this metadata out, but let me know if you want me to do that after merge.